pkg/restapi/rp/operation/operations.go

```diff
@@ -404,6 +443,20 @@ func (o *Operation) peekInvitationData(id string) *invitationData {
 	return o.transientInvitationData[id]
 }
 
+func (o *Operation) setThidInvitationData(d *thidInvitationData) {
+	o.thidInvDataLock.Lock()
+	defer o.thidInvDataLock.Unlock()
+
+	o.thidInvitationDataMap[d.threadID] = d
```

Comment:
Shouldn't this be in some store instead of local map ?

Reply:
@rolsonquadras it's on my TODO list: #29